plugins/operators/annotation.py

```
@@ -150,10 +154,154 @@ def config(self):
+            )
+
+        # Validate values for components that require them
+        component = attr_schema.get("component")
+        if component in values_components or attr_type in list_types:
+            values = attr_schema.get("values")
+            if values is not None:
+                if not isinstance(values, list):
+                    raise ValueError(
+                        f"Values for attribute '{attr_name}' must be a list, "
+                        f"got {type(values).__name__}"
+                    )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)